includes/transformer/class-comment.php

```diff
@@ -7,6 +7,7 @@
 
 namespace Activitypub\Transformer;
 
+use WP_HTML_Tag_Processor;
```

Comment:
```suggestion
```

It doesn't look like that's used anywhere?

Reply:
oops... I experimented with that ;)